src/bundle/Resources/views/admin/search/search.html.twig

```diff
@@ -0,0 +1,32 @@
+{% extends "EzPlatformAdminUiBundle::layout.html.twig" %}
+
+{% trans_default_domain 'search' %}
+
+{% block pageTitle %}{% endblock %}
+
+{% block content %}
+    <div id="react-udw"></div>
```

Comment:
Should UDW be a part of search page?

Reply:
This is something we should change I think. We load UDW module in every page (in layout.html.twig) but the container is added in some subpage. I think we should move this container to the `layout.html.twig` and remove it from every subpage.

Reply:
This is good point @dew326 . We should make a follow-up task on this UDW load.